cli/cmd/util.go

```diff
@@ -0,0 +1,88 @@
+package cmd
+
+import (
+	"crypto/rand"
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/cncf/servicebroker/k8s/service_controller/model"
+)
+
+// TODO(vaikas): Move these into a ../lib or ../../lib?
+
+// from: https://play.golang.org/p/4FkNSiUDMg
+// newUUID generates a random UUID according to RFC 4122
+func NewUUID() (string, error) {
```

Comment:
Removed it, it's not used yet. Once we need uuids, that's a good one, I did see that before too, just forgot to remove this since I didn't end up needing UUIDs after all.